ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/snippet/types/VariableDeclarationSnippet.java

```diff
@@ -106,4 +107,12 @@ private void addIdentifier(Token token) {
             foundVariableIdentifiers.add(new QuotedIdentifier(unescapedIdentifier));
         }
     }
+
+    /**
+     * Returns syntax node kind.
+     */
+    public SyntaxKind getSyntaxKind() {
```

Comment:
This is somewhat misleading. Since the syntax kind returned here is not the syntax kind of the variable decl itself. Instead of doing this, how about changing this method to return whether this var decl is declared using `var` or not? e.g., `isDeclaredWithVar()`

Reply:
Updated in a67b26d2ae21b4ddcd565b9584468933ead64dc1